src/main/resources/update/gitbucket-core_4.35.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<changeSet>
+  <!--================================================================================================-->
+  <!-- ACCOUNT_HIGHLIGHTER -->
+  <!--================================================================================================-->
+  <createTable tableName="ACCOUNT_HIGHLIGHTER">
```

Comment:
If we will have per-user preferences mechanism in GitBucket, it might be better to define a table like `ACCOUNT_PREFERENCES` or `USER_PREFERENCES` so that we can add columns for another per-user preference items to this table later.

By the way, GitBucket currently uses the browser's localStorage to store a line wrapping setting in the online editor. We may be able to use the same method for highlighter setting by putting a pulldown on the top of the source code viewer without adding a table.

I can't conclude which is a better solution immediately. I just want to let you know we have another option to store per-user settings for now.

Reply:
@takezoe 
> GitBucket currently uses the browser's localStorage to store a line wrapping setting in the online editor. We may be able to use the same method for highlighter setting by putting a pulldown on the top of the source code viewer without adding a table.

I didn't know this system... 😅 that could be choice too.

hmm.. IMHO user preferences table might be useful considering that we probably want to add more user preferences and if it does not belongs to a browser, setting reproduce regardless user's browser and user's device.

But As I said before I didn't know browser storage, so I'm really not sure which is better...

Reply:
I think we need to sort out what settings should belong to browser or user. Line wrapping setting may also need to be migrated to user preferences if we will introduce the per-user preferences table.

For now, could you update the table schema in case of storing further per-user preferences in the future? Then, I will test it and consider whether we should migrate other preferences currently use localStorage to the per-user preferences table.

Reply:
@takezoe I updated table schema and services.  Thank you for your consideration.

Reply:
Thanks! I will try this PR.